client/src/app/primitives/__tests__/TabLinksSpec.js

```diff
@@ -396,6 +396,53 @@ describe('<TabLinks>', function() {
     // in the current test environment
     it.skip('should set <small> selector on resize', function() {
 
+      // given

```

Comment:
Since this test is skipped, do we need a dedicated issue describing what to do, in order to not skip such tests in the future anymore? (Could e.g., be something for spring cleaning)?

Reply:
I think we should follow the guideline here to not have refactoring issues in our backlog. I recently [time boxed this](https://github.com/camunda/camunda-modeler/pull/2655#issuecomment-1004893229) without a sufficient solution. The comments are there, next time somebody stumbles above it, one can again try to find something in a time-boxed manner.

Reply:
:+1: